app-frontend/src/app/components/mapContainer/mapContainer.html

```diff
@@ -1,4 +1,22 @@
 <div class="map-container">
+  <div class="lab-control-panel">
+    <button class="btn btn-default"
+            ng-click="$ctrl.fitToBounds()"
+    >
+      <i class="icon-resize-full"></i>
+    </button>
+    <div class="filler"></div>
+    <button class="btn btn-default"
+            ng-click="$ctrl.zoomIn()"
+    >
+      +
+    </button>
+    <button class="btn btn-default"
+            ng-click="$ctrl.zoomOut()"
+    >
```

Comment:
Actually, I think the controls being added here was a mistake. I was going to move the map controls into the map components and out of the service but decided against it at some point and this got left over.